themes/powerline/powerline.theme

```diff
@@ -129,14 +129,15 @@ _lp_powerline_full_theme_activate() {
     # Load default colors if not already defined
     [[ -z ${POWERLINE_BATTERY_COLOR[@]+x}              ]] && POWERLINE_BATTERY_COLOR=(-1 238 0 0 -1 0)
     [[ -z ${POWERLINE_CHROOT_COLOR[@]+x}               ]] && POWERLINE_CHROOT_COLOR=(219 30 0 0 7 4)
+    [[ -z ${POWERLINE_CONTAINER_COLOR[@]+x}            ]] && POWERLINE_CONTAINER_COLOR=("${CONTAINER_NEUTRAL_COLOR[@]}")
```

Comment:
`CONTAINER_NEUTRAL_COLOR` does not exist.

Reply:
Fixed in two ways:  

1.  Moved the definition of POWERLINE_NEUTRAL_COLOR up higher so we could use it as a default
2. Changed  the name of the variable to be the correct one.